controller/src/main/java/io/pravega/controller/store/stream/InMemoryStream.java

```diff
@@ -823,6 +824,21 @@ private void handleStreamMetadataExists(final long timestamp, CompletableFuture<
         return result;
     }
 
+    @Override
+    CompletableFuture<Void> createSubscribersDataIfAbsent(StreamSubscribersRecord data) {
+        return null;
```

Comment:
please implement or throw notimplemented exception

Reply:
ok... this got missed. will implement.